src/main/java/stream/flarebot/flarebot/FlareBot.java

```diff
@@ -55,17 +55,78 @@
+import stream.flarebot.flarebot.commands.moderation.RolesCommand;
+import stream.flarebot.flarebot.commands.moderation.SetPrefixCommand;
+import stream.flarebot.flarebot.commands.moderation.WelcomeCommand;
+import stream.flarebot.flarebot.commands.moderation.mod.BanCommand;
+import stream.flarebot.flarebot.commands.moderation.mod.ForceBanCommand;
+import stream.flarebot.flarebot.commands.moderation.mod.KickCommand;
+import stream.flarebot.flarebot.commands.moderation.mod.MuteCommand;
+import stream.flarebot.flarebot.commands.moderation.mod.TempBanCommand;
+import stream.flarebot.flarebot.commands.moderation.mod.TempMuteCommand;
+import stream.flarebot.flarebot.commands.moderation.mod.UnmuteCommand;
+import stream.flarebot.flarebot.commands.moderation.mod.WarnCommand;
+import stream.flarebot.flarebot.commands.moderation.mod.WarningsCommand;
+import stream.flarebot.flarebot.commands.music.DeleteCommand;
+import stream.flarebot.flarebot.commands.music.JoinCommand;
+import stream.flarebot.flarebot.commands.music.LeaveCommand;
+import stream.flarebot.flarebot.commands.music.LoadCommand;
+import stream.flarebot.flarebot.commands.music.LoopCommand;
+import stream.flarebot.flarebot.commands.music.MusicAnnounceCommand;
+import stream.flarebot.flarebot.commands.music.PauseCommand;
+import stream.flarebot.flarebot.commands.music.PlayCommand;
+import stream.flarebot.flarebot.commands.music.PlaylistCommand;
+import stream.flarebot.flarebot.commands.music.PlaylistsCommand;
+import stream.flarebot.flarebot.commands.music.RandomCommand;
+import stream.flarebot.flarebot.commands.music.RepeatCommand;
+import stream.flarebot.flarebot.commands.music.ResumeCommand;
+import stream.flarebot.flarebot.commands.music.SaveCommand;
+import stream.flarebot.flarebot.commands.music.SearchCommand;
+import stream.flarebot.flarebot.commands.music.SeekCommand;
+import stream.flarebot.flarebot.commands.music.ShuffleCommand;
+import stream.flarebot.flarebot.commands.music.SkipCommand;
+import stream.flarebot.flarebot.commands.music.SongCommand;
+import stream.flarebot.flarebot.commands.music.SongNickCommand;
+import stream.flarebot.flarebot.commands.music.StopCommand;
+import stream.flarebot.flarebot.commands.random.AvatarCommand;
+import stream.flarebot.flarebot.commands.secret.ChangeAvatarCommand;
+import stream.flarebot.flarebot.commands.secret.DisableCommandCommand;
+import stream.flarebot.flarebot.commands.secret.EvalCommand;
+import stream.flarebot.flarebot.commands.secret.GuildCommand;
+import stream.flarebot.flarebot.commands.secret.LogsCommand;
+import stream.flarebot.flarebot.commands.secret.QueryCommand;
+import stream.flarebot.flarebot.commands.secret.QuitCommand;
+import stream.flarebot.flarebot.commands.secret.ShardRestartCommand;
+import stream.flarebot.flarebot.commands.secret.TestCommand;
+import stream.flarebot.flarebot.commands.secret.UpdateCommand;
+import stream.flarebot.flarebot.commands.secret.UpdateJDACommand;
```

Comment:
These should be Star as we use every command, just go against IntelliJ as it's dumb